salt/metalk8s/roles/bootstrap/init.sls

```diff
@@ -1,2 +1,14 @@
-Deploy bootstrap:
-  test.succeed_without_changes
+include:
+  - metalk8s.node.grains
+  - metalk8s.kubernetes.kubelet.standalone
+  - metalk8s._internal.preflight
+  - metalk8s.registry.populated
+  - metalk8s.repo.installed
+  - metalk8s.salt.master
+  - metalk8s.roles.minion
```

Comment:
If you include this, shouldn't you include it in all other roles (since it is a dependency)?

Don't we want `roles.node` as well? I guess this could replace the three lines above:

```diff
 include:
-  - metalk8s.node.grains
-  - metalk8s.kubernetes.kubelet.standalone
-  - metalk8s._internal.preflight
+  - metalk8s.roles.node
```

And do it for `roles.etcd` too?

Reply:
We can't here as the order is important. 
We need:
- the node grain at the very beginning
- then we deploy a standalone kubelet to be able to deploy registry, repo and salt-master
...

The `metalk8s.roles.node` configure entierly kubelet (not standalone) so we can't at this stage

Reply:
Also, there's a distinction between `minion` and `node`:

- `minion` is a node that is known to Salt. I.e. it's got `salt-minion` running (and upgraded)
- `node` is a node that's configured to be part of the Kubernetes cluster, i.e. it has `kubelet`, client certs,...